### ISSUE TITLE
Integrate UnKnoT to generate training proposals

### ISSUE THREAD

**Issue by mzur** [biigle/maia#57]
Offer [UnKnoT](https://github.com/BiodataMiningGroup/unknot) as an alternative to novelty detection or existing annotations in the same dataset to generate training proposals. This requires another annotated dataset and both the dataset to process with MAIA and the other annotated dataset having the "distance to sea floor" information for each image.

**Comment by mzur** [comment]
Also include the optimizations for Mask R-CNN training that were included in the UnKnoT configuration (e.g. improved augmentation).

**Comment by mzur** [comment]
Notes:
- There will be the following options to get training data in a MAIA job:
   - Novelty Detection (if there are interesting objects on a rather uniform background)
   - Existing Annotations (in the same dataset), this can't be combined with novelty detection any more
   - UnKnoT (if both datasets have distance to ground information)
- The configuration to train head and all layers will be replaced by a customizable "train scheme". Default scheme (layers, epochs, leraning rate):
   1. heads, 10, 0.001
   2. heads, 10, 0.0005
   3. heads, 10, 0.0001
   4. all, 10, 0.0001
   5. all, 10, 0.00005
   6. all, 10, 0.00001
